pipeline/src/util.rs

```diff
@@ -48,9 +47,9 @@ pub fn try_read_poly_set<P: PolySet, T: FieldElement>(
         .collect();
 
     (!column_names.is_empty()).then(|| {
-        let fname = format!("{name}_{}", P::FILE_NAME);
+        let path = dir.join(P::FILE_NAME);
```

Comment:
why remove `name`?

Reply:
Because these files are now written inside the backend. Either `EStarkDump` or `EStarkPolygon`. Backend already has output_dir. I think think we don't need another identifier string for the files.